src/zino/tasks/linkstatetask.py

```diff
@@ -39,10 +40,13 @@ class LinkStateTask(Task):
     case: Descriptions are mandated for both physical ports and their sub-units.
     """
 
+    sysuptime: Optional[int] = 0
```

Comment:
What's the difference between sysuptime set to 0 and None?

Reply:
`None` would usually mean that we don't know the `sysUpTime` value, while `0` would mean that the device booted *just now*.

However, the `Optional` here seems to be redundant. Within this plugin, the value isn't optional.  The original link state code basically defaults to using 0 as the base for calculating timestamps if the sysuptime value couldn't be fetched.